src/pages/Employees/components/EmployeeTable/EmployeeTable.tsx

```diff
@@ -42,75 +48,146 @@ export default function EmployeeTable({
     }
   };
 
+  const [showInactiveEmployees, setShowInactiveEmployees] = useState(false);
+
+  return (
+    <>
+      <TeamMemberSwitch
+        onChange={(e) => setShowInactiveEmployees(e.target.checked)}
+        isChecked={showInactiveEmployees}
+      />
+      <Suspense fallback={<EmployeeCardSkeleton />}>
+        <DeleteConfirmationModal
+          key={employeeToDelete ? employeeToDelete.id : undefined}
+          employee={employeeToDelete}
+          setEmployee={setEmployeeToDelete}
+          destroyEmployee={destroyEmployee}
+        />
+        <EmployeeModal
+          isOpen={!isEmpty(employeeToEdit)}
+          onClose={() => setEmployeeToEdit(null)}
+          onSave={(employee) => submitUpdateEmployee(employee as Employee)}
+          employee={employeeToEdit ? employeeToEdit : undefined}
+        />
+        <Box {...props}>
+          <EmployeeTableView
+            setEmployeeToEdit={setEmployeeToEdit}
+            setEmployeeToDelete={setEmployeeToDelete}
+            showInactiveEmployees={showInactiveEmployees}
+            useEmployees={useEmployees}
+          />
+        </Box>
+      </Suspense>
+    </>
+  );
+}
+
+function TeamMemberSwitch({
+  onChange,
+  isChecked,
+}: {
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  isChecked: boolean;
+}): JSX.Element {
+  return (
+    <FormControl
+      display="flex"
+      alignItems="center"
+      justifyContent="end"
+      marginTop="2em"
+    >
+      <FormLabel htmlFor="showInactiveEmployees" mb="0">
+        Show inactive team members
+      </FormLabel>
+      <Switch
+        id="showInactiveEmployees"
+        isChecked={isChecked}
+        onChange={onChange}
+      />
+    </FormControl>
+  );
+}
+
+interface EmployeeTableViewProps {
+  showInactiveEmployees: boolean;
+  setEmployeeToDelete: Dispatch<SetStateAction<Employee | null>>;
+  setEmployeeToEdit: Dispatch<SetStateAction<Employee | null>>;
+  useEmployees: typeof useEmployeesDefault;
+}
+
+function EmployeeTableView({
+  showInactiveEmployees,
+  setEmployeeToDelete,
+  setEmployeeToEdit,
+  useEmployees,
+}: EmployeeTableViewProps) {
+  const employeesFetched = useEmployees({ include: "skills", sort: "name" });
+
+  const employees = useMemo(
+    () =>
+      employeesFetched.filter((emp) =>
```

Comment:
not entirely sure, but i think this is another instance where we want to be doing this filtering on the server instead